### PR TITLE
week 2 homework

### PR DESCRIPTION
**Pull request by MichaelReyes1**
# Submitting Week 2 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [ X] Career Readiness
- [ X] Coding Challenges
- [ X] Coding Critique
- [X ] Coding Concepts
- [X ] Interview Questions

### ✋ Reminder

Do NOT merge your assessment!

### 📝 Feedback

If you would like specific feedback please leave details here:

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -13,39 +13,94 @@
...
+//ReferenceError: divisibleByThree is not defined- good failure
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
 
+//input: objects
+//output: divisible by three or not
+const divisibleByThree = (obj) => {
+//use dataype number
```
Comment (chrisyhlee) [inline code-challenges.test.js]: What do you mean by data type here? This line is using dot notation to access the value of the "number" key in the object.

File: code-challenges.test.js [hunk 2]
```diff
@@ -13,39 +13,94 @@
...
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
+describe('capFirstLetter', () => {
+  it('takes in an array of words and returns an array with all the words capitalized', () => {
+    const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+    // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+    const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+    // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+    expect(capFirstLetter()).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+    expect(capFirstLetter()).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
```
Comment (chrisyhlee) [inline code-challenges.test.js]: There should be arguments in each of these function calls

**Review by chrisyhlee — CHANGES_REQUESTED** [review]
I'm seeing confusion in terminology when you're explaining concepts and code. For example, in coding-concepts 4, you said a function is being invoked, but there is no function there. The question uses myCodingSkills, which is just an object, not a function. Another example is in your interview questions, where you are giving a barebones answer and not going in depth. Focus on working on this moving forward. This assessment displays Novice. In order to get an approval, you need to complete code challenge 2 then push those changes up.

File: code-challenges.test.js (hunk 1 above)
Comment (MichaelReyes1) [reply code-challenges.test.js]: i was thinking the number as a data type but i understand the dot notation just couldnt word it right.

File: code-challenges.test.js (hunk 2 above)
Comment (MichaelReyes1) [reply code-challenges.test.js]: i didnt get to finish this one , do i finish it and re push everything?

**Pull request closed without merging**